package.json

```diff
@@ -47,5 +49,11 @@
     "run-sequence": "^1.0.2",
     "script-loader": "^0.6.1",
     "webpack": "^1.8.3"
+  },
+  "dependencies": {
+    "suitcss-components-arrange": "^0.6.2",
+    "suitcss-components-button": "^4.0.0",
+    "suitcss-components-grid": "^2.0.2",
+    "suitcss-utils": "^0.10.3"
```

Comment:
But `postcss` is part of the build and is a dev dependency. The build couldn't run correctly without `suitcss`...right? It would fail because those modules wouldn't be there for the imports to work...right?